module/ffi_build.py

```diff
@@ -244,3 +247,20 @@
 
 ffi = FFI()
 ffi.cdef(CDEF)
+
+if ('XCFFIB_API_MODE' in os.environ and
+        int(os.environ['XCFFIB_API_MODE']) == 1):
+    ffi.set_source_pkgconfig(
+        '_xcffib',
+        ['xcb'],
+        """
+        #include "xcb/xcb.h"
+        #include "xcb/xproto.h"
+        #include "xcb/xevie.h"
+        #include "xcb/xcbext.h"
+        #include "xcb/render.h"
+        """
+    )
+
+    if __name__ == "__main__":
+        ffi.compile(verbose=True)
```

Comment:
This file rename looks like an API break, which will break downstream users, so we should figure out some way to avoid that.

Reply:
due to the move?  There was some oddity in that the ffi py name was shadowed by the ffi object in the module - which is why the rename - it seems like no one would have been able to access ffi.py without some module import path import lib magic.  But I would definitely like it to be as compatible as possible.  I believe in __init__.py I pull that back in as ffi in non api mode to make the change transparent externally

Reply:
Yeah, that sounds good to me, thanks.